feat: add sdk skill and cli (#2738)
Frontify/fondue: packages/sdk/adapters/claude-skill/skill/SKILL.md

Reviewer: see https://github.com/Frontify/skills/blob/main/docs

---
name: fondue

> description: Queries the locally installed `@frontify/fondue/sdk` to surface accurate Fondue component names, props, design tokens, utility classes, and setup guides — pinned to the version the project depends on. Use when picking a Fondue component (button, dialog, dropdown, tabs, …), finding a design token or utility class, or setting up the `@frontify/fondue` package.

Reviewer: ```suggestion
description: Queries the locally installed `@frontify/fondue/sdk` to surface accurate Fondue component metadata. ALWAYS use when picking a Fondue component (button, dialog, dropdown, tabs, …), finding a design token or utility class, or setting up the `@frontify/fondue` package.
```

Claude is set up to only accept 1% of context to be filled up by skills descriptors. That sounds like a lot at first but if you have like 10 skills or so it is very quickly overloaded and then claude truncates things, leading to very poor pick up rates. So we have to be as economical as possible with the description

- If the design calls for a value with no matching token, surface that explicitly — don't silently hardcode. The right answer is usually a token the user didn't know existed.

---

## Common pitfalls

| Mistake | Fix |
| ------- | --- |
| Calling `.where()` / `.get()` on the array returned by `list()` / `where()` | Arrays are arrays. Use native `.filter` / `.find`, or navigate back to a facet to query. |
| Assuming a component exists ("there must be a `Combobox`") | `components.has('Combobox')` first, or `components.where({ text: 'combobox' })` |
| Hardcoding a hex / px value in custom code | `tokens.where({ text: '<intent>' })` — most "obvious" values have a token |
| Treating icons like normal components, reading `status` / `props` | Detect with `node.category().name === 'icon'`. Icons have empty `status`, `props`, `related`. |
| Importing `Button` from the wrong path | `components.get('Button')?.importStatement` is authoritative |
| Suggesting setup steps from memory | `guides.get('getting-started')?.content` — always the live text |
| `console.log(node)` instead of `JSON.stringify` | Facet methods serialize as `[Function]`; always stringify before logging |

## Going deeper

> [`reference.md`](./reference.md) sits next to this file and ships with it — the formal contract: every export, every filter clause per domain, every node and facet type, error semantics. Load it when you need to verify a filter clause is valid (`ComponentFilter`, `TokenFilter`, `GuideFilter`) or that a method exists. Prefer it over web lookups or upstream GitHub.

Reviewer: ```suggestion
[`reference.md`](./reference.md) is the formal contract: every export, every filter clause per domain, every node and facet type, error semantics. Load it when you need to verify a filter clause is valid (`ComponentFilter`, `TokenFilter`, `GuideFilter`) or that a method exists. Prefer it over web lookups or upstream GitHub.
```
